main/src/main/scala/org/clulab/serialization/DocumentSerializer.scala

```diff
@@ -366,9 +370,25 @@ class DocumentSerializer extends LazyLogging {
       os.println(START_RELATIONS + SEP + relations.length)
       relations foreach (t => saveRelationTriple(t, os))
     }
+    // Added by Enrique to support sections names
```

Comment:
Do we have to be concerned about the `JSONSerializer` not knowing about the sections?  Perhaps at least a comment could be placed with the `sections` field to alert people that only the `DocumentSerializer` is aware of it so far.

Reply:
@kwalcock I  see there are two Json serialization classes:

- `org.clulab.serialization.json.JSONSerializer`
-` org.clulab.serialization.json4StopGraph.JSONSerializer`

Are both of them still relevant? 
I can't find the deserialization part for neither of them. I suspect they're abandoned in favor of the `DocumentSerializer`

Reply:
IDK.  The output could be used by a separate project that we don't know about.

Reply:
I think both serializations need to be supported for now...

Reply:
@kwalcock thanks!
@MihaiSurdeanu do you know if both json serializers are relevant today? And if not, which one is it?

Reply:
I know @myedibleenso implemented both (or one?) of them, and he was using them. @myedibleenso: do you have an answer to @enoriega's question?